aptible/validation.go

```diff
@@ -39,3 +39,16 @@ func validateURL(i interface{}, k string) (_ []string, errors []error) {
 
 	return
 }
+
+func errorsToWarnings(validator func(i interface{}, k string) ([]string, []error)) func(i interface{}, k string) ([]string, []error) {
```

Comment:
minor thing that could help make this a bit more readable is: 

```go
type ValidatorFunc func(i interface{}, k string) ([]string, []error)
type ValidatorFuncCallback func(i interface{}, k string) ([]string, []error)
func errorsToWarnings(validator ValidatorFunc) ValidatorFuncCallback {
```

with a brief explanation (i think above is correct)

Reply:
Good call. Terraform provides a type for this already so I should just do that.